doc/source/conf.py

```diff
@@ -79,6 +79,7 @@
 # directories to ignore when looking for source files.
 exclude_patterns = ['nb/dev']
 
+nbsphinx_allow_errors = True
```

Comment:
I’d like to be very strict here, because any errors that slip through will make it to the published documentation.

Reply:
It is still strict: I've added the `pytest --nbval` step, so we're still having the notebooks checked, but it provides more information: instead of just failing on the first failed notebook during the documentation build, it will run each of them as test and give the output.

Reply:
@Sheshuk I understand about the additional information by `pytest --nbval`; but I don’t think it’s necessary to allow errors here in `conf.py` to get that additional information, is it?

If _building the docs_ errors out if we have a broken notebook, then it will also error out in readthedocs, so we’re guaranteed never to successfully build broken documentation.
If building the docs doesn’t error out and we have rely on a different step in this CI workflow to alert us of any issues, then there’s always the risk of us overriding that failed test, or modifying this workflow without adjusting the docs build, or … and ending up with documentation that’s broken without us realising it.